### ISSUE TITLE
No v0 tag - usage instructions could be better

### ISSUE THREAD

**Issue by timsearle** [TWiStErRob/net.twisterrob.ghlint#274]
### Reproduction steps

1. See the "GitHub Actions" section under https://ghlint.twisterrob.net/usage/
2. Note that it only provides an out-of-context example of using GHLint
3. It references using the `v0` tag

### Expected behavior

That I could copy this suggestion as-is to my repository and use it

### Actual behavior

- I had to flesh out the rest of the workflow - which given GH-Lint is often likely to be in a standalone workflow due to a path filter on workflows/yaml files, could be more developer friendly.
- Once I did flesh out the workflow, e.g.:

```yaml
name: "Pull Request"

on:
  pull_request:
    branches:
      - main
    paths:
      - .github/workflows/**

jobs:
  gh-lint:
    name: "GH-Lint"
    runs-on: ubuntu-latest
    timeout-minutes: 3
    permissions:
      contents: read # actions/checkout
      security-events: write # codeql-action/upload-sarif
    steps:
      - name: "Checkout ${{ github.ref }} in ${{ github.repository }}"
        uses: actions/checkout@v4

      - name: "Run validation"
        id: ghlint
        uses: TWiStErRob/net.twisterrob.ghlint@v0

      - name: "Publish 'GH-Lint' GitHub Code Scanning analysis"
        uses: github/codeql-action/upload-sarif@v3
        with:
          sarif_file: ${{ steps.ghlint.outputs.sarif-report }}
```

I was greeted with:

> Error: Unable to resolve action `twisterrob/net.twisterrob.ghlint@v0`, unable to find version `v0`
